src/test/java/com/easyrest/tests/owner/ActionWithEmployeeTest.java

```diff
@@ -37,12 +39,13 @@ public void goToSignInOwner() {
     public void addWaiterTest() {
         createNewWaiterFacade = new CreateNewWaiterFacade(driver);
         createNewEmployee = new CreateNewEmployee(driver);
```

Comment:
just check if you use this object: createNewEmployee = new CreateNewEmployee(driver);

Reply:
Thanks